prestoadmin/collect.py

```diff
@@ -209,10 +210,10 @@ def system_info():
 
 def get_system_info(download_location):
 
-    if not exists(TMP_PRESTO_DEBUG):
-        run("mkdir " + TMP_PRESTO_DEBUG)
+    if not exists(TMP_PRESTO_DEBUG_REMOTE):
```

Comment:
but why was it already created as root? May be leftover from some other code?  Shouldn't all these be created with the same user as this is part of collect logs collect system_info